src/controller.ts

```diff
@@ -69,7 +69,7 @@ function init(step:types.Step):void {
 let timer: SetIntervalAsyncTimer;
 
 // eslint-disable-next-line unicorn/prevent-abbreviations
-async function startContainer(image:string, stepId:number, env?: [string]) : Promise<number> {
+async function startContainer(image:string, stepId:number, env: string[]) : Promise<number> {
```

Comment:
I think I changed this line yesterday. Did you want to revert my change, or was it lost in the merge ?